SPECS/kernel/kernel.spec

```diff
@@ -220,6 +223,12 @@ install -vdm 700 %{buildroot}/boot
 install -vdm 755 %{buildroot}%{_defaultdocdir}/linux-%{uname_r}
 install -vdm 755 %{buildroot}%{_prefix}/src/linux-headers-%{uname_r}
 install -vdm 755 %{buildroot}%{_libdir}/debug/lib/modules/%{uname_r}
+
+install -d -m 755 %{buildroot}%{_sysconfdir}/sysconfig
+install -c -m 644 %{SOURCE5} %{buildroot}/%{_sysconfdir}/sysconfig/cpupower
+install -d -m 755 %{buildroot}%{_unitdir}
```

Comment:
nit: I think these two lines to make the dir, install in said dir can be combined by using the -D option.

install -D -c -m 644 %{SOURCE6} %{buildroot}%{_unitdir}/cpupower.service